### PR TITLE
chore(storybook): link to swc on docs pages

### PR DESCRIPTION
**Pull request by castastrophe**
## Description

Add links to the Spectrum Web Components implementations for components that have them. These links are being added to the resource section and will be leveraging the web component technology logo (rather than Adobe) as discussed & approved by the SWC team.

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Validation steps

- [ ] Expect to see a link to the web component implementation for Accordion in the resource section: [link](https://pr-3356--spectrum-css.netlify.app/preview/?path=/docs/components-accordion--docs)

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:

- [ ] The pages render correctly, are accessible, and are responsive.

2. If components have been modified, VRTs have been run on this branch:

- [ ] VRTs have been run and looked at.
- [ ] Any VRT changes have been accepted (by reviewer and/or PR author), or there are no changes.

## To-do list

- [x] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [x] I have updated relevant storybook stories and templates.
- [ ] I have tested these changes in Windows High Contrast mode.
- [ ] If my change impacts **other components**, I have tested to make sure they don't break.
- [ ] If my change impacts **documentation**, I have updated the documentation accordingly.
- [x] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

File: .storybook/assets/index.css [hunk 1]
```diff
@@ -66,6 +67,15 @@ nav .spectrum-Site-logo {
 	padding: 20px !important;
 }
 
+/* Hide that first divider line in the top navigation */
+div.sb-bar > div > div > span:first-child {
```
Comment (cdransf) [inline .storybook/assets/index.css:71]: Is there a more stable way to select this? It feels a bit fragile. ✨

File: .storybook/assets/index.css [hunk 2]
```diff
@@ -66,6 +67,15 @@ nav .spectrum-Site-logo {
...
+	display: none;
+}
+
+div.sb-bar > div > div > div > button {
```
Comment (cdransf) [inline .storybook/assets/index.css:75]: Same concern as above. 🙂

**Review by cdransf — APPROVED** [review]
Some (minor) questions about selectors, but looks great!

File: .storybook/assets/index.css (hunk 1 above)
Comment (castastrophe) [reply .storybook/assets/index.css:71]: Oh it's insanely fragile but it's basically all we have access to in Storybook without overwriting the base components they're using. All the classes are randomly generated.

**Review by marissahuysentruyt — COMMENTED** [review]
Yay this is looking so great! It's so exciting seeing the teams coming together so closely now!

I wondered whether a few of the SWC components match up to differently-named components on the CSS side. Let me know what you think.

CSS modal- would that be something like SWC's dialog wrapper? or overlay?
CSS stepper - I believe this would be SWC's number field.

**Comment by castastrophe** [comment]
> Yay this is looking so great! It's so exciting seeing the teams coming together so closely now!
> 
> I wondered whether a few of the SWC components match up to differently-named components on the CSS side. Let me know what you think.
> 
> CSS modal- would that be something like SWC's dialog wrapper? or overlay? CSS stepper - I believe this would be SWC's number field.

Yeah, you're probably right but I think let's start with the obvious line-ups and we can always add others later on.

**Review by marissahuysentruyt — APPROVED** [review]

**Pull request merged**